### ISSUE TITLE
Incorrect q axis in WPIntegrator in PyHyperScattering version 0.2.0

### ISSUE THREAD

**Issue by PriyankaKetkar** [usnistgov/PyHyperScattering#84]
Bringing up a bug found in PyHyper version 0.2.0.  In 0.2.0, the q values of scattering curves simulated using NRSS are shifted lower than they should be, and switching to version 0.1.8 or 0.1.10 solved the issue.  Bug is likely in the Warp Polar Integrator, which shifts q values by a factor of sqrt(2).  More context below if needed.

I initially was running into a problem that my NRSS simulation Bragg peaks (red) were at lower q values than my experimental RSoXS primary Bragg peak (black).  It was confirmed that experimental RSoXS primary Bragg peak is matching with the AFM spacing after fourier transforming (gray line).  It also was confirmed that I am entering the voxel size (PhysSize = 1.95312 nm) correctly into the NRSS simulation and also the PyHyper reader.  After switching to PyHyper version 0.1.8 or 0.1.10, the red curve shifted to the right and now lines up with the black curve.

![image](https://user-images.githubusercontent.com/122052170/232608795-0653864c-98cc-4c50-8edf-e21c78386673.png)

**Comment by phonghnguyen** [comment]
My guess is that it has to do with these lines of code in WPIntegrator:

```python        

        if self.MACHINE_HAS_CUDA:
            TwoD = self.warp_polar_gpu(img_to_integ,center=(center_x,center_y), radius = np.nanmax(img_to_integ.shape))
        else:
            TwoD = skimage.transform.warp_polar(img_to_integ,center=(center_x,center_y), radius = np.nanmax(img_to_integ.shape))
```
The q-axis seem to be implicitly generated up to the given radius and this takes the radius to be the maximum dimension of the resulting simulated qx, qy image. The actual radius for a square image should be proportionally sqrt(2)/2 as per this drawing:

<img src="https://user-images.githubusercontent.com/31559388/232618132-1e25b7e1-155a-4392-a8b7-9ff72cf983fa.jpg" alt="IMG_4844" width="300" height="500">

Multiplying by sqrt(2)/2 would only work for square qx, qy images. A suitable fix that maintains functionality with non-square detector images might be something along these lines:
```python   
        if self.MACHINE_HAS_CUDA:
            TwoD = self.warp_polar_gpu(img_to_integ,center=(center_x,center_y), radius = np.sqrt((img_to_integ.shape[0] - center_x)**2 + (img_to_integ.shape[1] - center_y)**2))
        else:
            TwoD = skimage.transform.warp_polar(img_to_integ,center=(center_x,center_y), radius = np.sqrt((img_to_integ.shape[0] - center_x)**2 + (img_to_integ.shape[1] - center_y)**2))
```

**Comment by pbeaucage** [comment]
This looks like it should work, thanks for figuring it out so quickly!  @phonghnguyen can you put this into a PR so that @PriyankaKetkar can test?

**Comment by pdudenas** [comment]
The current fix looks to have the [default behavior](https://github.com/scikit-image/scikit-image/blob/5e74a4a3a5149a8a14566b81a32bb15499aa3857/skimage/transform/_warps.py#L1086-L1088) when radius is left as `None`. Should we implement this as a kwarg instead of hardcoded?

**Comment by pbeaucage** [comment]
I believe this was fixed in #85.  If not, please feel free to reopen.
